docs/release-notes/7.16.asciidoc

```diff
@@ -0,0 +1,54 @@
+[discrete]
+[[release-notes-7.16.0]]
+== 7.16.0
+
+[discrete]
+[[features-7.16.0]]
+==== Features
+* Adds the ability to configure trusted applications on a per-policy basis, allowing security administrators to target a set of hosts with specific configurations and settings. For example, trusted applications could be tailored to certain functions within an organization or for testing and troubleshooting purposes ({pull}112182[#112182], {pull}111051[#111051], {pull}110966[#110966])
+* Adds memory threat protection for macOS and Linux systems.
+* Provides certified applications for {sn} Security Operations (SecOps) and {sn} IT Service Management (ITSM), and introduces a new {sn} IT Operations Management (ITOM) connector ({pull}105440[#105440], {pull}114125[#114125]).
+* Adds logic to finding pending isolation or release statuses for endpoints added in {es-sec} >=v7.16 ({pull}115441[#115441]).
+* Adds {Fleet} actions and responses to the endpoint activity log and enriches the log by showing successful or failed action responses that were completed when the endpoint finished executing the action request ({pull}114905[#114905]).
+* Adds host risk metadata to alert details ({pull}113274[#113274]).
+* Resolve deep link IDs ({pull}111984[#111984]).
+* Allows users to create host isolation exceptions ({pull}111253[#111253]).
+* Cases can be imported and exported as saved objects ({pull}110148[#110148]).
+* Adds the Risky Hosts view to the Overview page ({pull}109553[#109553]).
+
+[discrete]
+[[bug-fixes-7.16.0]]
+==== Bug fixes and enhancements
+* The *Analyze event* option has been moved from the overflow menu to the Actions column. It now only displays events that can be opened in the visual event analyzer ({pull}115478[#115478]).
```

Comment:
Add for context? I'm not 100% certain if this is just on the Alerts page, though.
```suggestion
* The *Analyze event* option has been moved from the overflow menu to the Actions column on the Alerts page. It now only displays events that can be opened in the visual event analyzer ({pull}115478[#115478]).
```

Reply:
Looks as though it was moved out of the overflow menu to the Actions column in the Alerts table, as well as the Events one. How about:
The *Analyze event* option has been moved from the overflow menu to the Actions column within the Alerts and Events tables.